src/components/Link.js

```diff
@@ -2,8 +2,9 @@ import {Link} from 'gatsby';
 import ExternalLink from './ExternalLink';
 
 export default props => {
-  const {href, to, ...rest} = props;
+  let {href, to, ...rest} = props;
```

Comment:
Were these changes needed?

Reply:
The security patch was related to an outdated version of some websocket package, which was a dependency of Gatsby's. I needed to updated Gatsby. Upon doing so, the internal link behavior broke. A link `event/[...rest]` no longer resolved to `https://community.amplify.aws/event/[...rest]`. Instead it resolved relative to the current page (aka, `https://community.amplify.aws/event/event/[...rest]`). I needed to safeguard against this behavior.